Makefile

```diff
@@ -19,14 +19,19 @@ pull:
 	done
 
 build:
-	govuk-docker build
+	bin/govuk-docker build
```

Comment:
Should we make this consistent in all the project specific Makefiles?

Reply:
Yeah that would be good. @barrucadu can you sneak that into https://github.com/alphagov/govuk-docker/pull/9?